packages/providers/inpage-providers-hub/src/floatingButton/index.tsx

```
@@ -411,26 +428,26 @@
-                  alignItems: 'center', 
-                  justifyContent: 'center' 
+              <div
+                style={{
+                  height: '16px',
+                  width: '16px',
+                  display: 'flex',
+                  alignItems: 'center',
+                  justifyContent: 'center'
                 }}
               >
                 <svg width="16" height="16" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)